group_manager/group_manager.py

```diff
@@ -3,13 +3,143 @@
 __copyright__ = 'Copyright (c) 2021, Utrecht University'
 __license__   = 'GPLv3, see LICENSE'
 
-from flask import Blueprint, render_template
+from flask import Blueprint, make_response, render_template, request
+
+import api
 
 group_bp = Blueprint('group_bp', __name__,
                      template_folder='templates',
-                     static_folder='static/group_manager')
+                     static_folder='static/group_manager',
+                     static_url_path='/static')
 
 
-@group_bp.route('/example')
+@group_bp.route('')
 def index():
-    return render_template('group_manager/example.html')
+    response = api.call('group_data', data={})
```

Comment:
api.call no longer requires data to be set explicitly. Not saying this needs changing, rather checking whether it was known.